bot/utility/utility.py

```diff
@@ -96,32 +95,45 @@ async def about(self, ctx: commands.Context):
-        """Handler for the `howto code` command.
+    async def howto(self, ctx: commands.Context, subcommand: Optional[str]):
+        """Handler for the `howto` command.
 
-        Explains how to do various things
+        The available subcommands of this Command Group post individual help messages explaining how to do various things regarding the Discord server itself or other often needed tasks.
 
         Args:
             ctx (discord.ext.commands.Context): The context in which the command was called.
-            what (Optional[str]): What the user wants to know about
+            subcommand (Optional[str]): The subcommand the user wants to invoke.
         """
-        subcommands = {
-            "code": ("How to format code with discord markdown", howto_handler.code),
-        }
-        if what in subcommands:
-            embed = subcommands[what][1]()
-        else:
-            if what is not None:
-                message = f"Ich kenne diesen Befehl leider nicht. Hast du vielleicht einen der folgenden gemeint?\n"
-            else:
-                message = "Eine Sammlung nützlicher Befehle:\n"
-            embed = discord.Embed(
-                title = "How to !howto",
-                color = constants.EMBED_COLOR_INFO,
-                description = message + "\n".join([f"`{each}`: {subcommands[each][0]}" for each in subcommands])
-            )
+        message = "Ich kenne diesen Befehl leider nicht. Hast du vielleicht einen der folgenden gemeint?\n" if subcommand else None
+
+        await ctx.send(message, embed=discord.Embed(
+            title="Available HowTos",
+            description="\n".join(["**- " + c.name + "**: " + c.description for c in self.howto.commands])
+        ))
+
+    @howto.command(name='code', description="Code in Nachrichten formatieren")
+    @command_log
+    async def howto_code(self, ctx: commands.context):
+        """Handler for the `howto code` subcommand.
+
+        Explains how to properly format code using Discords code blocks (https://support.discord.com/hc/en-us/articles/210298617-Markdown-Text-101-Chat-Formatting-Bold-Italic-Underline-).
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+        """
+        embed = discord.Embed(
+            title="Code-Formatierung 📝",
+            color=11945797,
+            description="Wenn du deinen Code mit anderen teilen willst, dann kannst du hierfür sogenannte Codeblöcke verwenden. Sie sorgen dafür, dass dein Code vom Rest deiner Nachricht visuell abgegrenzt wird und macht ihn lesbarer für alle.\n\nFür einen __Inline-Codeblock__, schreibe deinen Code zwischen zwei Backticks (**`**).\nFür einene __Multi-Line-Codeblock__, verwende stattdessen jeweils drei (**```**).\n\nIm letzteren Fall kannst du außerdem nach den ersten Backticks mit einem Kürzel die jeweilige Programmiersprache angeben, um so sogar passendes Syntax-Highlighting zu erhalten.",
```

Comment:
Line is very long, pylint should actually warn at line length of 240 symbols. This is over 500. 
Do some line breaks to split up the string.